packages/zosjobs/__tests__/__system__/SubmitJobs.system.test.ts

```diff
@@ -221,13 +206,23 @@ describe("Submit Jobs - System Tests", () => {
             );
             expect(job.jobid).toBeDefined();
             expect(job.jobname).toBeDefined();
-            await deleteJob(job);
+            testEnvironment.resources.jobs.push(job);
         });
-
         it("should return the job info of a submitted JCL string", async () => {
-            const job: any = await SubmitJobs.submitJclString(REAL_SESSION, "//JOBNAME1 JOB", {jclSource: "stdin"});
+            const jobOrSpoolFiles = await SubmitJobs.submitJclString(REAL_SESSION, "//JOBNAME1 JOB", { jclSource: "stdin" });
+
+            // Check if the returned value is an array of spool files or a single job object
+            if (Array.isArray(jobOrSpoolFiles)) {
+                throw new Error("Expected a single job object, but received spool files.");
+            }
```

Comment:
I like being verbose here with the error message.
I'm curious if jest isn't too mad about not handling promise-rejections.
Another alternative could be to add an `expect` and let jest handle the error.